test/upstream-e2e-tests.sh

```diff
@@ -22,7 +22,7 @@ teardown_serverless || failed=1
 # Run upgrade tests
 if [[ $TEST_KNATIVE_UPGRADE == true ]]; then
   (( !failed )) && install_serverless_previous || failed=3
-  (( !failed )) && run_knative_serving_rolling_upgrade_tests || failed=4
+  (( !failed )) && run_rolling_upgrade_tests "${UPGRADE_TEST_SCOPE:-serving}" || failed=4
```

Comment:
I dont' see this variable to be set anywhere, so we're running just Serving upgrade tests by default?

Reply:
Yes. As we talked about this. When I finish this I could set it to `serving,eventing`, in order to see how much that will prolong the test wall time.

Reply:
Ok. We got one run with just Serving. Perhaps you could enable eventing in the next run and we'll see.